File: _query_ent1.js
const sqlite3 = require('better-sqlite3');
const path = require('path');

const dbPath = path.join(__dirname, 'databases', 'clean_database', 'clean.db');
const db = new sqlite3(dbPath);

const row = db.prepare("SELECT * FROM questions WHERE idQuestion = 'ENT1'").get();
if (row) {
    console.log("=== ENT1 Question Details ===\n");
    for (const [key, value] of Object.entries(row)) {
        console.log(`${key}:`);
        if (key === 'list') {
            try {
                const parsed = JSON.parse(value);
                console.log(JSON.stringify(parsed, null, 2));
            } catch (e) {
                console.log(value);
            }
        } else {
            console.log(value);
        }
        console.log();
    }
}
db.close();

File: _query_ent1.py
import sqlite3
import json

conn = sqlite3.connect('databases/clean_database/clean.db')
conn.row_factory = sqlite3.Row
cur = conn.cursor()
cur.execute("SELECT * FROM questions WHERE idQuestion = 'ENT1'")
row = cur.fetchone()
if row:
    print("=== ENT1 Question Details ===\n")
    for key in row.keys():
        value = row[key]
        if key == 'list':
            print(f"{key}:")
            try:
                parsed = json.loads(value)
                print(json.dumps(parsed, indent=2))
            except:
                print(value)
        else:
            print(f"{key}: {value}")
        print()
conn.close()
